models/experimental/functional_squeezebert/README.md

```diff
@@ -0,0 +1,19 @@
+## functional_squeezebert Demo
+## How to Run
+
+Use `pytest --disable-warnings --input-path="models/experimental/functional_squeezebert/demo/input_data.json" models/experimental/functional_squeezebert/demo/demo.py::test_demo[models.experimental.functional_squeezebert.tt.ttnn_functional_squeezebert-squeezebert/squeezebert-uncased]` to run the demo.
```

Comment:
Please put clear indications on what TT platforms (GS, WH N150, WH N300) these demos run on.

Refer to other demo READMEs for what I'm talking about, particularly in `models/demos/wormhole`.